python_modules/dagster/dagster/_core/definitions/asset_check_spec.py

```diff
@@ -61,6 +61,7 @@ class AssetCheckSpec(
             ("asset_key", PublicAttr[AssetKey]),
             ("description", PublicAttr[Optional[str]]),
             ("additional_deps", PublicAttr[Optional[Iterable["AssetDep"]]]),
+            ("blocking", PublicAttr[bool]),
```

Comment:
Considered whether this should be `is_blocking` instead. Though an advantage of "blocking" is that we could later accept an enum or other object that specifies more complex blocking behavior, if we needed it.

Thoughts on this @schrockn ?

Reply:
We are a bit inconsistent on this in our public API, but generally we do not do an "is" or "has" prefix (e.g. `passed`, `skippable`, 

![Screenshot 2024-01-29 at 6 06 11 AM](https://github.com/dagster-io/dagster/assets/28738937/647a374c-c3be-4a99-ad0c-2169e3d8dcfc)

Given that `blocking` is my default choice. I also like the optionality that you mention.